cpp/src/metrics/silhouette_score_batched_single.cu

```diff
@@ -0,0 +1,44 @@
+
```

Comment:
Just a little nitpick- we're referring to single as `float` in the filenames in many other places (including raft). It would be nice to stay consistent here.